mtp_cashbook/templates/base.html

```diff
@@ -1,56 +1,12 @@
-{% extends 'layouts/django/internal.html' %}
+{% extends 'mtp_base.html' %}
 {% load i18n %}
 
-{% block page_title %}{% trans "Digital cashbook" %}{% endblock %}
+{% block page_title %}{% trans "Digital Cashbook" %}{% endblock %}
 
-{% block proposition %}{% trans "Digital cashbook" %}{% endblock %}
+{% block proposition %}{% trans "Digital Cashbook" %}{% endblock %}
 
-{% block stylesheets %}
-  {% load staticfiles %}
-  {% if DEBUG %}
-  <link rel="stylesheet" href="{% static "stylesheets/app.css" %}" media="screen, handheld, projection, tv">
-  <link rel="stylesheet" href="{% static "stylesheets/app-print.css" %}" media="print">
-  {% else %}
-  <link rel="stylesheet" href="{% static "stylesheets/app.min.css" %}" media="screen, handheld, projection, tv">
-  <link rel="stylesheet" href="{% static "stylesheets/app-print.min.css" %}" media="print">
-  {% endif %}
-{% endblock %}
-
-{% block include_js %}
-  {% if DEBUG %}
-  <script src="{% static "javascripts/app.js" %}"></script>
-  {% else %}
-  <script src="{% static "javascripts/app.min.js" %}"></script>
-  {% endif %}
-  <script type="text/javascript">$(moj.init);</script>
-{% endblock %}
-
-{% block global_subheader %}
-  {% if request.user.is_authenticated %}
-  <div class="user-details">
-    {% blocktrans with full_name=request.user.get_full_name|default:request.user.username %}
-    Logged in as {{ full_name }}
-    {% endblocktrans %}
-    <span>
-      <a href="{% url 'logout' %}">{% trans "Sign out" %}</a>
-    </span>
-    <span>
-      <a href="#" class="js-FeatureTour-start" id="tour-resume">{% trans "Help" %}</a>
-    </span>
-  </div>
-  {% endif %}
-
-  <p class="subheader-item phase-banner">
-    <strong class="phase-tag phase-tag-alpha">{% trans "Alpha" %}</strong>
-    {# #TODO - add current location for feedback view #}
-    {% blocktrans %}This is a new service – your <a href="#">feedback</a> will help us to improve it.{% endblocktrans %}
-  </p>
-
-  {% if messages %}
-  <ul class="messages" role="alert" tabindex="-1">
-    {% for message in messages %}
-      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
-    {% endfor %}
-  </ul>
-  {% endif %}
+{% block user_menu %}
+  <span>
+    <a href="#" class="js-FeatureTour-start" id="tour-resume">{% trans "Help" %}</a>
```

Comment:
Did we already talk about this only being added by JS? I forget… Anyway: if it's a JS-only feature, I'd argue we should only be adding the start link by JS.